azure-pipelines.yml

```diff
@@ -63,16 +69,13 @@ stages:
       stageDisplayName: 'Build:'
       jobs:
         - job: build
-          displayName: 'Build, UT, analyze on SonarCloud and stage to repox'
+          displayName: 'Build and stage to repox'
           workspace:
             clean: all
           variables:
-            solution: 'SonarScanner.MSBuild.sln'
             sonarCloudOrganization: 'sonarsource'
             sonarCloudMsBuildProjectKey: 'sonarscanner-msbuild'
             sonarCloudMsBuildProjectName: 'SonarScanner for .NET'
-            buildPlatform: 'Any CPU'
-            buildConfiguration: 'Release'
             MAVEN_CACHE_FOLDER: $(Pipeline.Workspace)/.m2/repository
             MAVEN_OPTS: '-Xmx3072m -Dmaven.repo.local=$(MAVEN_CACHE_FOLDER)'
```

Comment:
Maybe this one can be dropped as well, but I am not sure if `variables` here are set as environment variable as well.

Reply:
They are used by both stages, so I moved them up to the globally accessible variables.